packages/client/screens/WalletSettingsScreen.tsx

```diff
@@ -26,6 +33,51 @@ export default function WalletSettingsScreen({ navigation, route }: SettingsStac
 
     return (
         <View style={styles.view}>
+            <Box
+                style={styles.walletDetailsWrapper}
+                backgroundColor={
+                    route.params.wallet.currencyType == "BITCOIN"
+                        ? "rgba(247, 147, 26, 0.25)"
+                        : "rgba(60, 60, 61, 0.25)"
+                }
+            >
+                <VStack style={styles.walletDetails}>
+                    <HStack justifyContent="space-between">
+                        <VStack>
+                            <Text>{route.params.wallet.name}</Text>
+                            <Box marginTop="2px"></Box>
+                            <Text size={"subheadline"} color={"text.500"}>
+                                {formatAddress(route.params.wallet.address)}
+                            </Text>
+                        </VStack>
```

Comment:
Can use the space prop here

```suggestion
                        <VStack space={"2px"}}>
                            <Text>{route.params.wallet.name}</Text>
                            <Text size={"subheadline"} color={"text.500"}>
                                {formatAddress(route.params.wallet.address)}
                            </Text>
                        </VStack>
```